git.go

```diff
@@ -52,17 +62,22 @@ func gitCommitAdd(c *Config) bool {
 	return cmd.ProcessState.Success()
 }
 
-func gitAdd(c *Config) bool {
-	cmd := exec.Command("git", "clone", c.Repo(), c.LocalRepoPath())
+func gitPull(pathToRepo string) bool {
+	cmd := exec.Command("git",
+		"-C",
+		pathToRepo,
+		"pull",
+		"--rebase",
+	)
 	cmd.Stdout = os.Stderr
 	cmd.Stderr = os.Stderr
 	cmd.Stdin = os.Stdin
 	cmd.Run()
 	return cmd.ProcessState.Success()
 }
 
-func gitClone(c *Config) bool {
-	cmd := exec.Command("git", "clone", c.Repo(), c.LocalRepoPath())
+func gitClone(repo string, path string) bool {
```

Comment:
nit: multiple args of the type same type can be written as `func gitClone(repo, path string)`